main.tex

```diff
@@ -29,6 +32,15 @@ \subsection{Sample subsection}
 
 
 \begin{thebibliography}{00}
+\bibitem{CarcassoneRules}
+Carcassone official rules: https://images.zmangames.com/filer\_public/d5/20/d5208d61-8583-478b-a06d-b49fc9cd7aaa/zm7810\_carcassonne\_rules.pdf

```

Comment:
This link doesn't work

Reply:
because it's formatted for latex. '_' character is an operator, so it need to be written '\_'